platform_versioned_docs/version-24.2/data_studios/index.mdx

```diff
@@ -30,6 +30,18 @@ Currently, Data Studios only supports AWS Batch compute environments that **do n
 
 Data Studios provides four container image templates: JupyterLab, RStudio Server, Visual Studio Code, and Xpra. The image templates install a very limited number of packages when the session container is built. You can install additional packages as needed during a session.
 
+The image template names include the version of the included analysis application and the Seqera image template version itself. The Seqera version string is of the following format:
+
+```ignore title="Image template version"
+<tool_version>-<seqera_version>-[u-<update_version>]
+```
+
+The following version strings apply:
```

Comment:
```suggestion
```

Reply:
I don't know if this suggestion comes through but I would leave out the introductory sentence.

Reply:
@justinegeffen, just a hanging list under the example?